.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "libdash"]
```

Comment:
I am wondering if instead of a submodule we just want to have a script that pulls libdash when we want to run the tests. Like a `setup_tests.sh`. Or somehow add a specific libdash library as a pip requirement but only for tests. The reason is that libdash is not an actual dependency of shasta and is only needed for tests, and if we do the same thing for both libdash and shasta, we will have a circular submodule chain (not sure if that can cause trouble).

Reply:
I removed the submodule and added another target in the Makefile test runner that downloads the tests, which solves this problem.

Reply:
Should this file be deleted too?